grails-app/controllers/org/pih/warehouse/api/LocalizationApiController.groovy

```diff
@@ -21,30 +21,41 @@ class LocalizationApiController {
     GrailsApplication grailsApplication
 
     def list() {
-        String languageCode = params.lang
+        String languageCode = params.languageCode
```

Comment:
Is this so we don't have conflicts with the spring locale interceptor?

Reply:
Yeah this triggers right now unnecessary language change when we are pulling translation after already doing the language change. Once we fully migrate to the changing locale with `lang` param this place will be adjusted accordingly, but for now this is unnecessary.